spec/models/manageiq/providers/redhat/infra_manager/vm_import_spec.rb

```diff
@@ -0,0 +1,73 @@
+describe ManageIQ::Providers::Redhat::InfraManager::VmImport do
+  let(:target_ems)  { FactoryGirl.create(:ems_redhat_with_authentication) }
+  let(:source_ems)  { FactoryGirl.create(:ems_vmware_with_authentication) }
+  let(:source_vm)   { FactoryGirl.create(:vm_with_ref, :ext_management_system => source_ems) }
+  let(:pool)        { FactoryGirl.create(:resource_pool) }
+  let(:source_host) { FactoryGirl.create(:host) }
+  let(:cluster)     { FactoryGirl.create(:ems_cluster) }
+  let(:storage)     { FactoryGirl.create(:storage_redhat) }
+
+  CLUSTER_ORIGINAL = 'Folder1/Folder @#$*2/Compute 3/Folder4/Cluster 5'.freeze
```

Comment:
Prefer `let` over constants here. Think this can cause issues with scope and redefining. /cc @kbrock correct me if I'm wrong